### ISSUE TITLE
Can't deploy Azure Postgres version 11

### ISSUE THREAD

**Issue by achalupa-ptc** [terraform-providers/terraform-provider-azurerm#3969]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform (and AzureRM Provider) Version

[root@salt /]# terraform -v
Terraform v0.12.5

Initializing provider plugins...
- Checking for available provider plugins...
- Downloading plugin for provider "azurerm" (terraform-providers/azurerm) 1.32.0...

### Affected Resource(s)

* `azurerm_postgresql_server`

### Terraform Configuration Files

```hcl
esource "azurerm_postgresql_server" "sc-postgres" {
  provider            = "azurerm.sc_postgres"

  name                = "sc-postgres-${var.deploy_type}-${var.ptc_region}"
  location            = "${data.azurerm_resource_group.rg.location}"
  resource_group_name = "${data.azurerm_resource_group.rg.name}"

  sku {
    name     = "GP_Gen5_4"
    capacity = 4
    tier     = "GeneralPurpose"
    family   = "Gen5"
  }

  storage_profile {
    storage_mb            = 102400
    backup_retention_days = 7
    geo_redundant_backup  = "Disabled"
  }

  administrator_login          = "postgres"
  administrator_login_password = "${var.postgres_password}"
  version                      = "11"
  ssl_enforcement              = "Enabled"
}
```

### Debug Output

Error: expected version to be one of [9.5 9.6 10 10.0 10.2], got 11.0

### Panic Output

N/A

### Expected Behavior

Azure supports postgres 11.  When will Terraform allow for it?

### Actual Behavior

Error: expected version to be one of [9.5 9.6 10 10.0 10.2], got 11.0

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

N/A

### References

N/A

* #0000

**Comment by francois-travais** [comment]
This is most probably a duplicate of #3327

**Comment by achalupa-ptc** [comment]
I agree except that ticket doesn't make reference to version 11.

**Comment by francois-travais** [comment]
I added a PR #3970 which fixes this issue but doesn't fix #3327, it legitimate this issue even more.
